CycloneDX/Services/PackagesFileService.cs

```diff
@@ -64,6 +63,7 @@ public async Task<HashSet<DotnetDependency>> GetDotnetDependencysAsync(string pa
                                 Name = reader["id"],
                                 Version = reader["version"],
                                 IsDevDependency = reader["developmentDependency"] == "true",
+                                IsDirectReference = true,
```

Comment:
Was was defaulting to `false` but packages in the config are always directly referenced by definition and would be migrated as those for `PackageReference`.